values.sample.yaml

```diff
@@ -144,6 +145,13 @@ tonicai:
 #   domain: <url-of-keycloak>
 #   realmId: <realm-id>
 
+#   Generic Saml SSO Config
+#   -----------------------
+#   provider: SAML
+#   metadataXml: either url or base64 must be provided, url is given precedence if both are provided
+#     url:
+#     base64:
```

Comment:
This is a different structure than the aws one. I didn't want to break existing chart values files, but this makes the two inconsistent which isn't great either.

1. Flatten this similar to AWS
2. Add backwards compat object to AWS and check for that or the existing base64 value when rendering aws sso

Reply:
My vote is for 2